modules/planning/tasks/deciders/path_bounds_decider.cc

```diff
@@ -84,6 +84,7 @@ Status PathBoundsDecider::Process(
 
   // Update the path boundary info to the frame.
   // TODO(all): update frame with path bound info.
+  reference_line_info->SetReachableS(std::get<0>(path_boundaries.back()));
```

Comment:
path_bound_decider hasn't been tested in simulation. If this would be the first time using anything from path_bound_decider, it will be great if simulation can be run before this change takes effect on regular ADC tests everyday.

Reply:
> path_bound_decider hasn't been tested in simulation. If this would be the first time using anything from path_bound_decider, it will be great if simulation can be run before this change takes effect on regular ADC tests everyday.

Also need to mention that this bound only used on fallback speed.